cmd/earth/main.go

```diff
@@ -870,6 +871,20 @@ func (app *earthApp) actionBuild(c *cli.Context) error {
 		secretsprovider.FromMap(secretsMap),
 		authprovider.NewDockerAuthProvider(os.Stderr),
 	}
+
+	var withSSH bool
+	sshPath := defaultSSHAuthSock()
```

Comment:
I would perhaps use the same SSH flag we had before to configure this as well. It allows the user the override the SSH socket if they want to.

The one defined as:

```
&cli.StringFlag{
			Name:        "ssh-auth-sock",
			Value:       defaultSSHAuthSock(),
			EnvVars:     []string{"EARTHLY_SSH_AUTH_SOCK"},
			Usage:       "The SSH auth socket to use for ssh-agent forwarding",
			Destination: &app.buildkitdSettings.SSHAuthSock,
		},
```

Reply:
great catch, this has been fixed now.